lib/diagnostic.js

```diff
@@ -7,22 +7,39 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min () {
+const min = function min (array) {
+  let result = array
 
+  for (let i = 1, max = array.length; i < max; i++) {
+    const current = array[i]
+    result = current > result ? current : result
+  }
+  return min()
```

Comment:
`min` is the name of your function, I think you actually want to return `result` here!